proof/refine/ARM/IpcCancel_R.thy

```diff
@@ -311,8 +312,10 @@ lemma blocked_cancel_ipc_corres:
                          and (\<lambda>_. (st = Structures_A.BlockedOnSend epPtr p
                                       \<longrightarrow> (\<exists>list. rv = Structures_A.SendEP list))
                                 \<and> (st = Structures_A.thread_state.BlockedOnReceive epPtr reply_opt p'
-                                     \<longrightarrow> (\<exists>list. rv = Structures_A.RecvEP list)))" and
-                      P'="?conc_guard and valid_ep' ep" in corres_inst)
+                                     \<longrightarrow> (\<exists>list. rv = Structures_A.RecvEP list)))"
+                  and P'="valid_objs' and valid_release_queue_iff and st_tcb_at' ((=) st') t
```

Comment:
?conc_guard was meant to match with all the concrete guards except sym_refs, and the intention was to make it easier to drop sym_refs here. So, there isn’t a need to make this change, if you had removed `and _` after ?cong_guard in the pattern above. But, of course, there is no need to revert what you did, either.